Files/Views/ModernShellPage.xaml.cs

```diff
@@ -26,6 +26,26 @@ public sealed partial class ModernShellPage : Page, IShellPage
     {
         public SettingsViewModel AppSettings => App.AppSettings;
 
+        public GridLength SidebarWidth
+        {
+            get
+            {
+                var localSettings = ApplicationData.Current.LocalSettings;
+                var sidebarWidth = 200d;
+
+                if (localSettings.Values.ContainsKey("SidebarWidth"))
+                {
+                    sidebarWidth = Math.Min(Math.Max((double)localSettings.Values["SidebarWidth"], 200), 500);
+                }
+
+                return new GridLength(sidebarWidth, GridUnitType.Pixel);
+            }
+            set
+            {
+                ApplicationData.Current.LocalSettings.Values["SidebarWidth"] = value.Value;
+            }
+        }
+
```

Comment:
Please use this format for saving the settings.
```
public bool OpenASpecificPageOnStartup
{
        get => Get(false);
        set => Set(value);
}
```

Reply:
Made changes.